backend/wmg/data/rankit.py

```diff
@@ -0,0 +1,36 @@
+import numpy as np
+import scipy.stats
+import scipy.sparse
+import numba as nb
+
+
+@nb.jit
+def quantiles(n: int) -> np.ndarray:
```

Comment:
I'm going to update this with Pablo's code to ensure we give the same score to equal values in a later pr